src/modelio.cs

```diff
@@ -3026,5 +3041,39 @@ interface MDLMatrix4x4Array : NSCopying {
 		[Export ("getDouble4x4Array:maxCount:")]
 		nuint _GetDouble4x4Array (IntPtr valuesArray, nuint maxCount);
 	}
+
+	[iOS (12,0), Mac (10,14), TV (12,0)]
+	[BaseType (typeof (MDLAnimatedValue))]
+	interface MDLAnimatedQuaternion {
```

Comment:
same situation - the type _documented_ as part of iOS 12.0
but all members are in 13.0
https://developer.apple.com/documentation/modelio/mdlanimatedquaternion/3237269-doublequaternionattime?language=objc